### PR TITLE
webui: Quit button does not work on the Live image

### PR DESCRIPTION
**Pull request by adamkankovsky**

### CODE REVIEW

**Review by KKoukiou — COMMENTED** [review]
This needs some explanation. ;)

**Review by KKoukiou — COMMENTED** [review]
I will ask @martinpitt if he can take a look into this one. As I am a bit noob with the best practices on bash scripting.

File: ui/webui/webui-desktop [hunk 1]
```diff
@@ -86,4 +86,9 @@ else
     sleep 3
 fi
 
-HOME="$BROWSER_HOME" $BROWSER http://127.0.0.90"$URL_PATH"
+trap 'kill $B_PID; exit 0' INT TERM
```
Comment (martinpitt) [inline ui/webui/webui-desktop:89]: The signal list is incomplete; you should at least also terminate on `EXIT`, but adding `QUIT` and `PIPE` can't hurt either.

Also, please drop the `exit 0` -- running into that trap *already* means that your script is exiting.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Certainly a little convoluted, but if the Quit button emits SIGTERM or so, then that's an okay way to kill the browser with it.

File: ui/webui/webui-desktop [hunk 2]
```diff
@@ -86,4 +86,9 @@ else
     sleep 3
 fi
 
-HOME="$BROWSER_HOME" $BROWSER http://127.0.0.90"$URL_PATH"
+trap 'kill $B_PID' INT TERM EXIT
```
Comment (martinpitt) [inline ui/webui/webui-desktop:89]: Oh, there's already a trap in line 76. I'm not actually sure if they are additive, or the second one would replace the first (my hunch is the latter?) Can you just move this kill into line 76 then?

Comment (adamkankovsky) [reply ui/webui/webui-desktop]: So I finally modified it to just one trap and created a cleanup function to make the code clearer.

**Review by martinpitt — APPROVED** [review]
Looks good now, cheers!

**Pull request merged**